explainaboard/processors/processor.py

```diff
@@ -516,12 +525,19 @@ def get_overall_statistics(
         custom_features: dict = metadata.get('custom_features', {})
         custom_analyses: list = metadata.get('custom_analyses', [])
 
-        metric_configs: dict[str, dict[str, MetricConfig]] = {
-            "example": metadata.get('metric_configs', {})
-        }
+        metric_configs = metadata.get("metric_configs")
+        if metric_configs is not None:
+            metric_configs_dict = {
+                "example": {
```

Comment:
Thanks for fixing this! Just to clarify, it looks like `metric_configs` can only be used to replace `example` level metrics. Metrics for other levels (e.g. span) will always be the default ones. Is this intentional?

Reply:
This keeps the original behavior, since metric_configs in metadata can only represent one analysis level. If we need to modify other levels, we need some additional syntax on metadata.

(I think this behavior could be resolved if we used analysis levels to only lookup default features and metrics. This is somewhat off-topic, and would be good to discuss on #518)

Reply:
Thanks! I think the current behavior is kind of unusual because users can override the `example` level but not the other ones. But it makes sense to leave it as it is for now because metadata doesn't support additional analysis levels.